default.nix

```diff
@@ -775,6 +775,10 @@ rec {
     sha256 = "sha256-debC8ZpbIjgpEeISCISU0EVySJvf+WsUkUaLuJ526wA=";
   };
 
+  tidbits = with nixpkgs; {
+    drun = drun;
+  };
+
```

Comment:
Why the extra `tidbits`? We already have plenty of tidbids in the toplevel, e.g.
```
inherit (nixpkgs) wabt wasmtime wasm
```
just add it there.

Reply:
Ah, okay, I didn't see that!